app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -133,6 +147,11 @@ class BloodPressureHistoryScreen(
     context.startActivity(intent)
   }
 
+  private fun displayNameGenderAge(name: String, gender: Gender, age: Int) {
+    val genderLetter = resources.getString(gender.displayLetterRes)
+    toolbar.title = resources.getString(R.string.patientsummary_name, name, genderLetter, age)
```

Comment:
We try to avoid using string resources across features even though the actual string values might be same. Could you please duplicate the string resource and name it scoped to this screen (ex: `bphistory_toolbar_title`)?